Mental illness/redme/readme.md

```diff
@@ -0,0 +1,42 @@
+# Mental Illness In Tech Industry
+
+## Tech’s quiet battle with depression
+
+The tech and startup world is built on the backs of incredibly bright minds. It's known for its innovation and resilience and a culture that fosters high-productivity.
```

Comment:
The Tech and startup world is built on the backs of incredibly bright minds. It's known for its innovation and resilience and a culture that fosters high-productivity. But it has a dark underbelly.

Everyone around the world has mental health, but not everyone talks about it.According to OSMI data (open source mental illness survey), 51% of tech professionals have been diagnosed with a mental health condition. By comparison, 19.1% of U.S. adults experience mental illness, according to the National Alliance on Mental Illness.